references/cli/def.go

```diff
@@ -295,7 +295,7 @@ func generateTerraformTypedComponentDefinition(cmd *cobra.Command, name, kind, p
 	}
 
 	switch provider {
-	case "aws", "azure", "alibaba", "tencent", "gcp", "baidu":
+	case "aws", "azure", "alibaba", "tencent", "gcp", "baidu", "ec":
```

Comment:
It's difficult to understand what it is `ec`. How about renaming it to `elastic` https://www.elastic.co/cloud/?